api/azure/utils.go

```diff
@@ -848,23 +848,41 @@ func (config *AzureProvider) ConfigReader(logging log.Logger, clusterType string
 	return nil
 }
 
+// checking if the environment variables are set or not.
 func setRequiredENV_VAR(logging log.Logger, ctx context.Context, cred *AzureProvider) error {
 	tokens, err := util.GetCred(logging, "azure")
 	if err != nil {
 		return err
 	}
```

Comment:
In another case if the user has not added the token to file then it will return err as no such file present

so we must check if the env is absent then try to get it from file

Reply:
Also that you need to handle error and return err
https://github.com/kubesimplify/ksctl/blob/15ac1baf8b987b4317070df1ab6cace41120c647/api/azure/main.go#L238

Reply:
also look where all it need to be added

Reply:
if anyone token is missing from the envitomenet variable get out of envioment variable check and try to read from file that will make things steam line ig

Reply:
@Manda-supraja26 no issues for now this pr is mergable to another branch and will fix and merge it to main branch 👍🏼